devito/finite_differences/finite_difference.py

```diff
@@ -117,7 +117,7 @@ def first_derivative(expr, dim, fd_order=None, side=centered, matvec=direct,
 
     This is also more easily obtainable via:
 
-    >>> (f*g).dx
+    >>> (f*g).dx.evaluate
```

Comment:
I wouldn't do .evaluate here, and show the Derivative repr instead.

Then, line below, I'd add 

```
The corresponding finite difference approximation can be obtained via

>>> (f*g).dx.evaluate
...
```

I'd add this pattern to all FD functions in this file

Reply:
Well no, this is the `first_derivative` function that always returns the FD. the `.dx` only is in `Derivative`